### ISSUE TITLE
cannot find ‘.csv’ file

### ISSUE THREAD

**Issue by Bao0319** [LeonChenUCR/ThermalGCN#1]
Hi！
I run the code in Windows. All dependencies are satisfied. 
I'm having trouble in running `python run.py`. Before I met the problem, I have run the `Generate.py`.

Traceback (most recent call last):
File "E:\ThermalGCN\ThermalGCN-main\dataset\run.py", line 24, in <module>
os.rename("./data/Edge.csv", EdgeFile)
FileNotFoundError: [WinError 2] 系统找不到指定的文件。: './data/Edge.csv' -> './data/Edge_0_0.csv'

I try to find the '.csv' file. But where is it ? And how can I get the '.csv' file.

**Comment by leonchenucr** [comment]
Thank you for your question. I think you should first create ./data/ folder, then run the run.py.

**Comment by Bao0319** [comment]
Thank you for your reply! I have solved the problem. 
It seems that the code `cmd = "../hotspot -c ./hotspot.config -f Chiplet_Core.flp -p Chiplet_Core"+str(i)+"_Power"+str(j)+".ptrace -steady_file Chiplet.steady  -model_type grid -detailed_3D on -grid_layer_file Chiplet.lcf -grid_steady_file Chiplet.grid.steady"` in `run.py`  does not work correctly under `Windows`.
I tried to run the code under `Ubuntu`, and this time there was no problem.
